packages/api/src/resolvers/home_feedback/index.ts

```diff
@@ -0,0 +1,100 @@
+import { DatabaseError } from 'pg'
+import { QueryFailedError } from 'typeorm'
+import { env } from '../../env'
+import {
+  ErrorCode,
+  HomeFeedback,
+  HomeFeedbackError,
+  HomeFeedbackSuccess,
+  MutationSendHomeFeedbackArgs,
+  SendHomeFeedbackError,
+  SendHomeFeedbackSuccess,
+} from '../../generated/graphql'
+import { analytics } from '../../utils/analytics'
+import { authorized } from '../../utils/gql-utils'
+import {
+  HomeFeedback as HomeFeedbackModel,
+  HomeFeedbackType,
+  HomeFeedbackType as HomeFeedbackTypeModel,
+} from '../../entity/home_feedback'
+import {
+  createHomeFeedback,
+  findHomeFeedbackByUserId,
+} from '../../services/home_feedback'
+
+export const sendHomeFeedbackResolver = authorized<
+  SendHomeFeedbackSuccess,
+  SendHomeFeedbackError,
+  MutationSendHomeFeedbackArgs
+>(async (_parent, { input }, { uid, log }) => {
+  const { author, site, subscription } = input
+  analytics.capture({
+    distinctId: uid,
+    event: 'send_home_feedback',
+    properties: {
+      feedback: input.feedbackType,
+    },
+  })
+
+  if (!author && !site && !subscription) {
+    log.error('author, site, or subscription is required')
+    return {
+      errorCodes: [ErrorCode.BadRequest],
+    }
+  }
```

Comment:
Probably better added the `sanitised` directive to the field

Reply:
@sywhb so I need to check if at least one is set, but others can be null. Can I do that with `sanitised`?